package.json

```diff
@@ -50,7 +50,8 @@
     "collectCoverageFrom": [
       "src/**/*.{js,jsx}",
       "!src/example/**",
-      "!src/index.js"
+      "!src/index.js",
+      "!src/flags/create.js"
```

Comment:
Considering the amount of input variables and the complexity of result output for `create` command, creating test for it would be a rather expensive undertaking.